go/vt/discovery/healthcheck.go

```diff
@@ -73,13 +73,11 @@ var (
 	tabletURLTemplate        *template.Template
 )
 
-// See the documentation for NewHealthCheck below for an explanation of these parameters.
 const (
+	// See the documentation for NewHealthCheck below for an explanation of these parameters.
```

Comment:
This should be outside the block. Otherwise, the comment will only be shown against the first value.